src/strings/PowerFxResources.en-US.resx

```diff
@@ -4760,4 +4760,12 @@
     <value>Unsupported untyped/JSON conversion type '{0}' in argument.</value>
     <comment>Error Message shown to user when a unsupported type is passed in type argument of AsType, IsType and ParseJSON functions.</comment>
   </data>
+  <data name="ErrReachedMaxJsonDepth" xml:space="preserve">
+    <value>Maximum depth reached while traversing {Locked=JSON} payload.</value>
```

Comment:
>{Locked=JSON} [](http://example.com/codeflow?start=50&length=13)

This doesn't need the locked directive (we aren't talking about the JSON function, but the JSON format, which may be translated in other languages).